src/Servers/Kestrel/Core/src/Internal/LoggerExtensions.cs

```diff
@@ -64,7 +64,7 @@ internal static class LoggerExtensions
 
         public static void FailedToLocateDevelopmentCertificateFile(this ILogger<KestrelServer> logger, string certificatePath) => _failedToLocateDevelopmentCertificateFile(logger, certificatePath, null);
 
-        public static void FailedToLoadDevelopmentCertificate(this ILogger<KestrelServer> logger, string certificatePath) => _failedToLoadDevelopmentCertificate(logger, certificatePath, null);
+        public static void FailedToLoadDevelopmentCertificate(this ILogger<KestrelServer> logger, string? certificatePath) => _failedToLoadDevelopmentCertificate(logger, certificatePath, null);
```

Comment:
Looking at FailedToLoadDevelopmentCertificate(), I don't see how this could be culled with a null certificate path. If it can be, we should probably fix the log message.